scripts/lib/dependencies.ts

```diff
@@ -192,16 +192,17 @@ export async function getPublishedReleaseTagNames(owner: string, repo: string) {
 
 // Dependency's that adhere to the following criteria may use this function
 // to get a list of available versions:
-// - The Dependency is hosted at a github repository.
-// - Versions are gathered from the tag that is on each github release.
+// - The dependency is hosted at a GitHub repository.
+// - Versions are gathered from the tag that is on each GitHub release.
 // - Versions are in semver format.
 export async function getPublishedVersions(githubOwner: string, githubRepo: string, includePrerelease: boolean): Promise<string[]> {
   const tagNames = await getPublishedReleaseTagNames(githubOwner, githubRepo);
-  const allVersions = tagNames.map((tagName: string) => tagName.replace(/^v/, ''));
+  let versions = tagNames.map((tagName: string) => tagName.replace(/^v/, ''));
 
+  versions = versions.filter(version => semver.valid(version) !== null);
   if (!includePrerelease) {
-    return allVersions.filter(version => semver.prerelease(version) === null);
+    versions = versions.filter(version => semver.prerelease(version) === null);
```

Comment:
Similarly, this reads better as:
```suggestion
    versions = versions.filter(version => !semver.prerelease(version));
```